partcad-cli/src/partcad_cli/click/commands/init.py

```
@@ -7,30 +7,148 @@
     "-p",
     "--private",
     is_flag=True,
+    default=False,
+    cls=DynamicPromptOption,
     help="Initialize this package as private",
+    prompt="Do you want this package to be private?",
     show_envvar=True,
 )
 @click.pass_context
-def cli(ctx: click.rich_context.RichContext, private):
+def cli(ctx: click.rich_context.RichContext, **kwargs):
```

Ruff (0.8.2)

`cli` is too complex (11 > 10)

(C901)